icefoganalytics/sfa-client
# Enhancement: Add Parent Dependents Sections to Student Application Endpoint Response

Relates to https://github.com/icefoganalytics/student-financial-aid/issues/2
Fixes https://github.com/icefoganalytics/sfa-client/issues/39

# Context

As part of the "view submitted applications and their current status" project, this task covers returning the parent dependents data.

# Data Format

```json
{
  "id": "some-application-id",
  "parentDependants": {},
}
```

# Testing Instructions

1. Boot the `sfa-client` repo via `API_PORT=3100 dev up`
2. Boot the `student-financial-aid` front- and back-ends.
3. Log in to the app at http://localhost:8080/
4. Go to http://localhost:3000/api/portal/students/8634/applications/1995 and you will see the empty parent dependents section.
5. Go to http://localhost:3000/api/portal/students/1720/applications/2013 and you will see some parent dependents values.
